Resources/Prototypes/Nyanotrasen/Entities/Clothing/Masks/masks.yml

```diff
@@ -28,3 +28,16 @@
     sprite: Nyanotrasen/Clothing/Mask/mask_oni_red.rsi
   - type: Clothing
     sprite: Nyanotrasen/Clothing/Mask/mask_oni_red.rsi
+
+- type: entity
+  parent: ClothingMaskPullableBase
+  id: felinidcone
```

Comment:
Casing for ID would be FelinidCone. The ID should also probably just be ClothingNeckFelinidCone and go in the neck slot, no?

Reply:
i intend it to block eating and drinking

Reply:
does ingestionblocker care about which slot it's in...?

Reply:
*shrugs*

Reply:
i fixed casing but it doesn't want me to close this oddly